docker-compose.yml

```diff
@@ -1,22 +1,6 @@
 version: '3.7'
 services:
 
-  php74:
-    build:
-      context: .docker/development/php74
-    working_dir: /application
-    stdin_open: true
-    volumes:
-      - .:/application:cached
-
-  php80:
```

Comment:
We removed 8.0 and 7.4, but didn't added 8.3

Reply:
I managed to create the new image, so now everything includes PHP 8.3 as well